src/devices/Ws2812b/Ws2812b.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using System.Drawing;
+
+namespace Iot.Device.Ws2812b
+{
+    /// <summary>
+    /// Driver for WS2812B. Intelligentcontrol LED integrated light source
+    /// </summary>
+    public class Ws2812b : IntegratedLed
+    {
+        /// <summary>
+        /// Spi clock frequency for WS2812B
+        /// </summary>
+        public const int SpiClockFrequency = 3000000;
+
+        private SpiDevice _spiDevice;
+        private readonly byte[] _buffer;
+
+        private const byte Bit0 = 0b0100; // 333ns low(dummy), 333ns high(T0H), 667ns low(T0L)
+        private const byte Bit1 = 0b1100; // 667ns high(T1H), 667ns low(T1L)
+        private const byte ResetDelay = 105; // 280us low
+
+        private const byte ColorSize = 4;
```

Comment:
I think `ColorBitSize` is more appropriate name, also please move it above consts defining bits so it's clearer they're related